tests/test-cluster.py

```diff
@@ -278,6 +278,24 @@ def test_calico_in_nodes(self):
                 assert False
             print("Calico found in node {}".format(vm.vm_name))
 
+    def test_calico_interfaces_removed_on_snap_remove(self):
+        """
+        Test that calico interfaces are not present on the node
+        when the microk8s snap is removed.
+        """
+        vm = VM(backend)
+        vm.setup(channel_to_test)
+        print("Waiting for machine {}".format(vm.vm_name))
+        vm.run("/snap/bin/microk8s.status --wait-ready --timeout 240")
+        while True:
```

Comment:
I would like us to have some sort of timeout here instead of waiting forever.

```suggestion
        while True:
```

Reply:
Thanks a lot for the feedback @neoaggelos ! I've added a timeout like you suggested (I'm using the same value of 240 seconds like it is passed on line 289).